docs/contributing/guidelines-for-writing-upgrade.md

```diff
@@ -2,6 +2,8 @@
 
 Keep in mind that upgrade instructions are written for users that do not understand our system well, so more clear they are, more useful they are.
 
+## Introduction
+
```

Comment:
I like instructions with with link to specific changes - https://github.com/shopsys/shopsys/blame/d315cfb96b9e9ed344e2b81c9f1deec7c89873b7/docs/upgrade/UPGRADE-unreleased.md#L13

Reply:
Note about the possibility to link specific changes added to upgrade guidelines.